platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -195,6 +195,13 @@ typedef void (^MGLOfflinePackRemovalCompletionHandler)(NSError * _Nullable error
  KVO change notifications on the shared offline storage object’s `packs` key
  path. Removals from that array result in an `NSKeyValueChangeRemoval` change.
  
+ @note When you remove an offline pack, any resources that are required by that
+    pack, but not other packs, become eligible for removal from the offline
+    database. Because the offline database is also used as a general purpose
```

Comment:
Let’s talk about “offline storage” instead of “the offline database”. I’ve tried to avoid the word “database” to head off any suggestions that we expose more sophisticated querying functionality.